machine/machine_core/machine_virtual.py

```diff
@@ -27,6 +27,8 @@
 import tempfile
 import sys
 import time
+from collections import namedtuple
```

Comment:
FTR: This should be available in RHEL 8/py 3.6, see https://docs.python.org/3/library/collections.html?highlight=namedtuple#collections.namedtuple